src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelDiscoveryDataAggregator.cs

```diff
@@ -27,6 +28,7 @@ public ParallelDiscoveryDataAggregator()
         IsAborted = false;
         TotalTests = 0;
         _metricsAggregator = new ConcurrentDictionary<string, object>();
+        DiscoveredExtensions = new Dictionary<string, HashSet<string>>();
```

Comment:
[cosmetic] Now that you have "concrete" type, you could simplify declaration here

```suggestion
        DiscoveredExtensions = new();
```

Reply:
Done.